CMakeLists.txt

```diff
@@ -194,10 +194,6 @@ endif ()
 set(ENTERPRISE_INCLUDE_DIR "enterprise")
 option(USE_ENTERPRISE "enable enterprise build" OFF)
 
-# we want the following definitions to be in effect for both rocksdb and arangodb
```

Comment:
note: these CMake variables are not necessary with the new version of RocksDB, and removed here intentionally.